compliance_checker/acdd.py

```diff
@@ -384,6 +451,237 @@ def check_time_extents(self, ds):
+        return ratable_result((len(keyword_readable),
+                              len(ds.keywords.split(','))),
+                              'keywords_readable', msgs=[])
+
 
 class ACDDNCCheck(BaseNCCheck, ACDDBaseCheck):
     pass
+
+class ACDD1_1Check(ACDDNCCheck):
+
+    _cc_spec_version = '1.1'
+    _cc_description = 'Attribute Conventions for Dataset Discovery (ACDD) 1.1'
+    register_checker = True
+
+    def __init__(self):
+        super(ACDD1_1Check, self).__init__()
+        self.rec_atts.extend(['keywords_vocabulary',
+                        ('cdm_data_type', ['Grid', 'Image', 'Point',
+                                           'Radial', 'Station', 'Swath',
+                                           'Trajectory'])])
+
+        self.sug_atts.extend(['publisher_name',       # publisher,dataCenter
+                                'publisher_url',        # publisher
+                                'publisher_email',      # publisher
+                                'geospatial_vertical_positive'
+                              ])
+
+
+class ACDD1_3Check(ACDDNCCheck):
+
+    _cc_spec_version = '1.3'
+    _cc_description = 'Attribute Conventions for Dataset Discovery (ACDD) 1.3'
+    register_checker = True
+
+    def __init__(self):
+        super(ACDD1_3Check, self).__init__()
+        self.high_rec_atts.extend([('Conventions',
+                                    self.verify_convention_version)])
+
+        self.rec_atts.extend(['geospatial_vertical_positive',
+                              'geospatial_bounds_crs',
+                              'geospatial_bounds_vertical_crs',
+                              'publisher_name',       # publisher,dataCenter
+                              'publisher_url',        # publisher
+                              'publisher_email',      # publisher
+                              'source'])
+
+        self.sug_atts.extend([  # 1.3.1, technically
+                ('creator_type', ['person', 'group', 'institution',
+                                  'position']),
+                'creator_institution',
+                ('cdm_data_type', ['Grid', 'Image', 'Point', 'Radial',
+                                   'Station', 'Swath', 'Trajectory']),
+                'platform',
+                # TODO: make dependent on platform
+                'platform_vocabulary',
+                'keywords_vocabulary',
+                'instrument',
+                'metadata_link',
+                'product_version',
+                'references',
+                ('publisher_type', ['person', 'group', 'institution',
+                                    'position']),
+                'instrument_vocabulary',
+                'date_metadata_modified',
+                'program',
+                'publisher_institution',
+            ])
+
+    def check_history(self, ds):
+        #@TODO Create a history check
+        return
+
+    def check_source(self, ds):
+        #@TODO Create a source check
```

Comment:
There already exists checks to see if these are present.  Don't think it checks if empty or not.